cmd/registry/main.go

```diff
@@ -22,6 +22,7 @@ import (
 	gorhandlers "github.com/gorilla/handlers"
 	"github.com/yvasiyarov/gorelic"
 	"golang.org/x/net/context"
+	_ "../../registry/auth/openshift"
```

Comment:
No need to fix this, but for future reference it's better to avoid relative paths in imports